src/views/CountryOperations/AppealOperationTable/index.tsx

```diff
@@ -110,7 +110,7 @@ function AppealOperationTable(props: Props) {
                 strings.appealsTableRequestedAmount,
                 // FIXME amount_requested should be number from server
                 (item) => (
-                    isTruthyString(item.amount_requested)
+                    item.amount_requested
```

Comment:
Let's use `isDefined` and lets remove the fixme and the Number cast if we're getting it as number